packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -31,24 +35,77 @@ const propTypes = {
 const defaultProps = {
   disableUnselectedRows: false,
   onChange: undefined,
+  onBlur: undefined,
   selectedIndexes: [],
 };
 
+const rowsSelector = 'row-group';
+
 class SelectableTableRows extends React.Component {
   constructor(props) {
     super(props);
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleOnBlur = this.handleOnBlur.bind(this);
+  }
+
+
+  componentDidMount() {
+    // Adds ARIA-Label for Pre-selected rows
+    const rowGroup = document.querySelector(`#${rowsSelector}`);
+    for (let rowIndex = 0; rowIndex < rowGroup.children.length; rowIndex += 1) {
+      const preselectedRow = rowGroup.children[rowIndex];
```

Comment:
Should this simply be called `row`? There is no guarantee that it is pre-selected at this point.

Reply:
it has been removed and being referred as row [here](https://github.com/cerner/terra-core/blob/cc6ee3fad6bea01cc24bb5ce812e6483291d781d/packages/terra-table/src/SelectableTableRows.jsx#L173)